src/utils/constants.ts

```diff
@@ -3,22 +3,30 @@ export const SECONDS_IN_YEAR = 365 * 24 * 60 * 60;
 
 export const Routes = {
   DASHBOARD: '/',
-  LOGIN: '/login',
-  REGISTER: '/register',
-  CREATE_EVENT: '/create-event',
-  EDIT_EVENT: '/edit-event',
-  EVENT: '/event',
-  USER: '/user',
+  LOGIN: '/prihlasit',
+  REGISTER: '/registrovat',
+  CREATE_EVENT: '/vytvorit-akci',
+  EDIT_EVENT: '/upravit-akci',
+  ATTEND_EVENT: '/zucastnit-se',
+  EVENT: '/sportovni-akce',
+  USER: '/uzivatel',
 };
 
 export const THEME = {
   LIGHT: 'light',
   DARK: 'dark',
 };
 
+export const PAGINATION = {
+  LIMIT: '8',
+  PAGE: '1',
+};
+
 export const ERROR_MESSAGE = {
   INVALID_CREDENTIALS: 'Nesprávné přihlašovací údaje.',
   GENERIC_ERROR: 'Vyskytla se chyba. Zkuste to znovu později.',
+  BAD_CREDENTIALS_EN: 'email or password is incorrect.',
```

Comment:
You should react to the HTTP status rather than the message. I should ideally return some code that will NEVER change in places where we need to be granular; the error message is more for you or for debugging. Let's go with the HTTP status for now. So 400 = email/psw error

Reply:
The problem with the message is that it can be translated, and suddenly, your logic would break. Let's avoid that.